### ISSUE TITLE
JetBrains: Make it easier to open files from search result

### ISSUE THREAD

**Issue by ryankscott** [sourcegraph/sourcegraph#37812]
In user testing we had people struggle to understand how to open the file from the results (and thinking it wasn't possible).

**Comment by sourcegraph-bot-2** [comment]
Heads up @muratsu @jjinnii @ryankscott - the "team/integrations" label was applied to this issue.

**Comment by philipp-spiess** [comment]
The native view adds a super obvious button to the bottom right, maybe this is something we should also consider?

![Image](https://user-images.githubusercontent.com/458591/176134380-1293917e-5943-4549-9d02-29c3cca8eadc.png)

**Comment by vdavid** [comment]
Yeah, excellent idea, we could add an "Open in Editor" / "Open in Browser" button in this location
